samples/startup-script/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "compute-sample-startup-script",
+  "version": "1.0.0",
+  "description": "Start a Google Compute Engine and run the startup script.",
+  "main": "index.js",
+  "dependencies": {
+    "@google-cloud/compute": "^0.10.0"
```

Comment:
Oh. That way we need to change the samples-test task in CircleCI to run `npm link ../../' in this folder, to make sure it runs against the source code that is checked out and not the source code from NPM.

Reply:
Thanks. Added an additional link to the `samples-test` script. Should I split it up in two scripts or is it OK for now?